components/brave_wallet_ui/components/desktop/account-list-item/index.tsx

```diff
@@ -52,11 +57,8 @@ export const AccountListItem = ({
     onClick(account)
   }, [onClick, account])
 
-  const removeAccount = React.useCallback(() => {
-    let confirmAction = confirm(`Are you sure to remove ${account.name}?`)
-    if (confirmAction) {
-      onRemoveAccount(account.address, isHardwareWallet, account.coin)
-    }
+  const removeAccount = React.useCallback(async () => {
```

Comment:
Is `async` needed here? No function is `await` in the function body.

Reply:
removed